src/realm/table.cpp

```diff
@@ -291,6 +291,36 @@ void Table::insert_column_link(size_t col_ndx, DataType type, StringData name, T
     get_descriptor()->insert_column_link(col_ndx, type, name, target, link_type); // Throws
 }
 
+size_t Table::add_column_list(DataType type, StringData name)
+{
+    size_t ndx = get_column_count();
+    insert_column_list(ndx, type, name);
+    return ndx;
+}
+
+void Table::insert_column_list(size_t column_ndx, DataType type, StringData name)
+{
+    DescriptorRef subdesc;
+    insert_column(column_ndx, type_Table, name, &subdesc);
+    switch (type) {
+        case type_Int:
+        case type_Bool:
+        case type_Float:
+        case type_Double:
+        case type_String:
+        case type_Binary:
+        case type_Timestamp:
+            subdesc->add_column(type, "list");
+            break;
+        case type_OldDateTime:
+        case type_Table:
```

Comment:
From a binding's perspective, lists-of-lists is already supported. But we wish to provide an easy way to support a list of integers. Currently, our users have to create a class with one property which is a bit cumbersome.